mzLib/MassSpectrometry/Enums/DissociationType.cs

```diff
@@ -109,6 +109,11 @@ public enum DissociationType
         /// </summary>
         LowCID,
 
+        /// <summary>
```

Comment:
if this is not needed in the PR, let's put it in another and see that such a mechanism has all the fragmentation type associate with it.